SMS/SMSMessages.cpp

```diff
@@ -173,11 +173,15 @@ TLMessage *SMS::parseTPDU(const TLFrame& TPDU)
 	LOG(DEBUG) << "SMS: parseTPDU MTI=" << TPDU.MTI();
 	// Handle just the uplink cases.
 	switch ((TLMessage::MessageType)TPDU.MTI()) {
-		case TLMessage::DELIVER_REPORT:
 		case TLMessage::STATUS_REPORT:
 			// FIXME -- Not implemented yet.
 			LOG(WARNING) << "Unsupported TPDU type: " << (TLMessage::MessageType)TPDU.MTI();
 			return NULL;
+		case TLMessage::DELIVER_REPORT: {
```

Comment:
Ah gotcha. So that I understand this diff, this allows us to handle deliver on smqueue? That wasn't enabled before I am guessing?